docs/omero_figure_scripting.rst

```diff
@@ -147,19 +147,19 @@ a heatmap of colors applied to Shapes on the figure panel.
 
 #.  Alternatively, perform the following steps:
 
-#.  For the Image you wish to use, add some ROIs to the Image. You can see the ROI and Shape IDs in the iviewer ROI table.
+#.  For the Image you wish to use, add some ROIs to the Image. You can see the ROI IDs in the iviewer ROI table.
 
-#.  To setup the OMERO.table, create a CSV file with an ``Roi`` column and a ``Shape`` column containing the corresponding IDs and 1
+#.  To setup the OMERO.table, create a CSV file with an ``Roi`` column containing the corresponding IDs and 1
```

Comment:
Now this is confusing.
I can have, according to the example given, more lines with the same ROI ID. 
I think it should be explained here that these lines correspond to different shapes under that ROI ?
But even if I understand this (if this is indeed the case), then it is not clear how do I make sure that the values pertain to the correct shape, as the Shape IDs are now gone from the csv ?

Reply:
Sorry, I missed that I'd still got duplicate ROI IDs in the table. Fixed in f66583e. Thanks